### PR TITLE
Player: Document "WaitForPutAway"

### PR DESCRIPTION
**Pull request by engineer124**
A system exists within player to delay running a certain function until after the item put away process is complete. This gets used when, for example, starting a conversation with an actor and player delays the talking action until after the item is put away. Docs are taken from OoT, see this PR:

https://github.com/zeldaret/oot/pull/1936

However, MM adds an additional functionality to run a cutscene during the item put-away process, although this is only used once during the get-item cutscene to adjust how the get-item cutscene was changed in MM

### CODE REVIEW

File: include/z64player.h [hunk 1]
```diff
@@ -884,8 +884,8 @@ typedef enum PlayerCueId {
 #define PLAYER_STATE1_200        (1 << 9)
 // 
 #define PLAYER_STATE1_400        (1 << 10)
-// 
-#define PLAYER_STATE1_800        (1 << 11)
+// Currently carrying an actor
+#define PLAYER_STATE1_ACTOR_CARRY (1 << 11)
```
Comment (fig02) [inline include/z64player.h:888]: I was wondering in oot if we wanted to go with `PLAYER_STATE1_CARRYING_ACTOR` instead. At least most of the flags i have named so far in oot are in "active" tense, as in it is what player is currently doing. But I think this is also fine 🤷

Comment (engineer124) [reply include/z64player.h:888]: `PLAYER_STATE1_CARRYING_ACTOR` does sound a little more natural to me, but what it is currently is also fine

Comment (fig02) [reply include/z64player.h:888]: https://github.com/zeldaret/oot/pull/2213

**Review by AngheloAlf — APPROVED** [review]

File: include/z64player.h (hunk 1 above)
Comment (fig02) [reply include/z64player.h]: this change was accepted in oot, if you want to change it here as well

Comment (fig02) [reply include/z64player.h]: oops seems like it already is changed

**Review by fig02 — APPROVED** [review]
This is a biased approval

**Review by hensldm — APPROVED** [review]

**Pull request merged**